pkg/kudoctl/resources/dependencies/resolve.go

```diff
@@ -103,18 +96,10 @@ func dependencyWalk(
 		// a relative dependency path must begin with either './' or '../'
 		isRelativePackage := strings.HasPrefix(childPackageName, "./") || strings.HasPrefix(childPackageName, "../")
 		if isRelativePackage {
-			if operatorDirectory == nil {
```

Comment:
There is no need in this whole dance, now that dependencies come with the scope-aware dependency resolver.